source/codegen/metadata/nidmm/CHANGES.md

```diff
@@ -97,12 +108,22 @@ with the session_repository.
 - `InitExtCal` : Added a 'custom_close' tag to this function, since this API has a corresponding close function called 'CloseExtCal'
 
 `'enum'` tag added to the following functions:
-- `'apertureTime`' parameter of function `'GetApertureTimeInfo'`
-- `'action`' parameter of function `'CloseExtCal'`
-- `'calType`' parameter of function `'CalibrationType'`
-- `'offsetCompOhms`' parameter of function `'ConfigureOffsetCompOhms'`
-- `'configuration`' parameter of function `'Control4022'`
-- `'type`' parameter of function `'CalAdjustMisc'`
-- `'powerLineFrequencyHz`' parameter of function `'ConfigurePowerLineFrequency'`
-- `'triggerDelay`' parameter of function `'ConfigureTrigger'`
-- `controlAction` parameter of function `'Control'`
```

Comment:
What is the reason for removing these enum tags?

Reply:
they are not removed. I just added more tags to the Changes.md. these are same as the once below. don't know why it is shown as deleted

Reply:
Yeah pull in latest changes from the feature branch and try to fix this issue